src/Microsoft.AspNetCore.SignalR/DefaultHubLifetimeManager.cs

```diff
@@ -104,7 +104,7 @@ public override Task InvokeGroupAsync(string groupName, string methodName, objec
         public override Task InvokeUserAsync(string userId, string methodName, object[] args)
         {
             return InvokeAllWhere(methodName, args, connection => 
-                string.Equals(connection.User.FindFirst(ClaimTypes.NameIdentifier).Value, userId, StringComparison.Ordinal));
+                string.Equals(connection.User.FindFirstValue(ClaimTypes.NameIdentifier), userId, StringComparison.Ordinal));
```

Comment:
Is it impossible that there will be no `NameIdentifier` claim? This was my first guess but than I decide that this could happen for anonymous connections and could cause NullReferenceException